samples/client/petstore/typescript-fetch/builds/default/src/apis/PetApi.ts

```diff
@@ -23,7 +24,7 @@ import {
 } from '../models';
 
 export interface AddPetRequest {
-    body: Pet;
+    pet: Pet;
```

Comment:
please don't run the `bin/openapi3/...` script, just the `bin/typescript-fetch-petstore-all.sh` and delete the  `inlineObject.ts` files

Reply:
Thanks! I have pushed a new commit to fix this